tests/test_codesyncd/buffer_handler.test.js

```diff
@@ -159,6 +170,69 @@ describe("bufferHandler", () => {
         return true;
     };
 
+    const generateMockTabs = (tabPath1 = tabFilePath1) => {
```

Comment:
Seems like unused variable? and where have we dofiend tabPath1 and tabFilePath2 that we are using in this function?

Reply:
This file is WIP, have to sort this out.

Regarding definition of tabFilePath1/tabFilePath2, we are defining them in `beforeEach`: https://github.com/codesyncapp/codesync-vs/blob/5504414598067ece350ea2465ea7e9ec8e888e91/tests/test_codesyncd/buffer_handler.test.js#L96